compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -101,6 +101,10 @@ static llvm::cl::opt<bool> clDispatchGenerateWorkloadRegion(
     "iree-flow-dispatch-generate-workload-region",
     llvm::cl::desc("Generate the workload region"), llvm::cl::init(true));
 
+static llvm::cl::opt<bool> clCollapseDimensions(
+    "iree-flow-form-dispatch-regions-collapse",
+    llvm::cl::desc("Collapse dimensions"), llvm::cl::init(true));
```

Comment:
we should probably check performance numbers if we want it default on. let me kick in the benchmark.

Please also improve the description of the flag.

Reply:
+1 for checking perf numbers. We do want it on by default. Ideally we should also remove the flag in the near future (or better not add it)